flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -366,7 +366,7 @@ function check_logs_for_errors {
       | grep -ic "error" || true)
   if [[ ${error_count} -gt 0 ]]; then
     echo "Found error in log files:"
```

Comment:
Can we add here that only a part of the log files are printed and that one should access the log files for a proper analysis?

Reply:
sure